datafusion/core/src/datasource/avro_to_arrow/schema.rs

```diff
@@ -137,15 +136,7 @@ fn schema_to_field_with_props(
                 .collect();
             DataType::Struct(fields?)
         }
-        AvroSchema::Enum(EnumSchema { symbols, name, .. }) => {
```

Comment:
This PR changes how DataFusion converts Avro's `Enum` to an Arrow type but I don't think it affects users because DataFusion currently doesn't support Avro's `Enum`.

Reply:
So the idea is that previously the code would try to map `Enum` to a `Dictionary` type, but as you point out there was no test coverage so it probably didn't work anyways